commands.json

```diff
@@ -1,4 +1,3 @@
 {
-  "commands": ["clear A8", "flag C3",
-    "flag D2", "clear H8"]
+  "commands": []
```

Comment:
Where it the first move, thought you gonna start it first. comeeeeeeeeeeeeeeeeeeeeeeeeee on1